MainMenu.cs

```diff
@@ -0,0 +1,60 @@
+using Godot;
+using System;
+using Godot.Collections;
+
+public class MainMenu : Control
+{
+	private Control _settingsMenu;
+	
+	private bool _disabled = false;
+	private bool _newplayer = false;
+	// Declare member variables here. Examples:
+	// private int a = 2;
+	// private string b = "text";
+
+	// Called when the node enters the scene tree for the first time.
+	public override void _Ready()
+	{
+		GetNode<Button>("VBoxContainer/StartButton").GrabFocus();
+		_settingsMenu = GetNode<Control>("SettingsMenu");
+		_settingsMenu.Visible = false;
+	}
+
+	//  // Called every frame. 'delta' is the elapsed time since the previous frame.
+	//  public override void _Process(float delta)
+	//  {
+	//      
+	//  }
+
+	private void OnStartButtonPressed()
+	{
+		if (_disabled)
+			return;
+
+		_disabled = true; // so multiple instances cant be hit
```

Comment:
Nitpick: Instead of commenting what the variable is here, you could comment on the variable itself and explain it there.